transactionoutbox-spring/src/test/java/com/gruelbox/transactionoutbox/acceptance/TransactionOutboxSpringDemoApplication.java

```diff
@@ -4,17 +4,12 @@
 import com.gruelbox.transactionoutbox.Persistor;
 import com.gruelbox.transactionoutbox.SpringInstantiator;
 import com.gruelbox.transactionoutbox.SpringTransactionManager;
-import com.gruelbox.transactionoutbox.SpringTransactionOutboxConfiguration;
 import com.gruelbox.transactionoutbox.TransactionOutbox;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.Lazy;
 
-@Import({
```

Comment:
I'm never sure what I'm looking at when it comes to spring changes. Is this just a simple revert or was the previous version incorrect? I guess either way it clearly works!

Reply:
This is a simple revert.
Nevertheless, we can omit this import here because the same import is present in [ExternalsConfiguration](https://github.com/gruelbox/transaction-outbox/blob/master/transactionoutbox-spring/src/test/java/com/gruelbox/transactionoutbox/acceptance/ExternalsConfiguration.java#L8) class, and `ExternalsConfiguration` itself is imported automatically (because it's in the same package as the main class)

Reply:
Perfect. That's all I needed to know.